bahmnicore-api/src/main/java/org/bahmni/module/bahmnicore/service/impl/PatientDocumentServiceImpl.java

```diff
@@ -85,6 +87,16 @@ public String saveDocument(Integer patientId, String encounterTypeName, String c
         }
     }
 
+    private String getBasePathByEncounterType(String encounterTypeName) {
+        String basePath;
+        if(encounterTypeName.equalsIgnoreCase(LAB_RESULT_ENCOUNTER_TYPE)){
+            basePath = BahmniCoreProperties.getProperty("bahmnicore.documents.lablitebaseDirectory");
+        } else{
+            basePath = getBasePath();
+        }
+        return basePath;
+    }
+
     private String getBasePath() {
         return BahmniCoreProperties.getProperty("bahmnicore.documents.baseDirectory");
```

Comment:
Can we move this line as well to the above function ? And we remove `getBasePath` function ?

Reply:
@mohan-13 getBasepath is being used in two other functions so did not remove that.